docs/source/resources/task/index.rst

```diff
@@ -1,7 +1,12 @@
 Supported Tasks
 ======================
 
+*To be completed by adding task discriptions.*
+
 .. toctree::
    :maxdepth: 1
 
-    Pour Water <pour_water.md>
\ No newline at end of file
+    Ramdom Reach
```

Comment:
Ramdom Reach is just a tutorial example for demonstration. It should not be included in the official env list